### PR TITLE
Add listing of requirements for community events.

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/organizer-guide/confs/community-events.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Community Events
+================
+
+We are happy to promote events that are created by our community.
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/community-events.rst]: created by our community > independently organized by members of our community

File: docs/organizer-guide/confs/community-events.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+They are **not** officially organized by Write the Docs organizers,
+but they are sister events that share a similar goal.
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/community-events.rst]: sister > sibling? (not even sure that the family association fits here, maybe "adjacent" or "friendly" or something else)

File: docs/organizer-guide/confs/community-events.rst [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Requirements
+------------
+
+In order to be listed as a community event,
+we require a few things.
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/community-events.rst]: general: any chance we can decide to use one-sentence-per-line? the line breaks are confusing when editing/reviewing

File: docs/organizer-guide/confs/community-events.rst [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+This is primarily to make sure people understand who is organizing the event,
+and to make sure companies can't run events inside our community for their own profit.
+
+Community events:
+
+* Must have a Code of Conduct
+* Must specify they **are not** official Write the Docs events, but must link back to Write the Docs as a refenence to the community
+* Must list the organizers, and they must be from multiple companies
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/community-events.rst]: and they must > who must

File: docs/organizer-guide/confs/community-events.rst [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Requirements
+------------
+
+In order to be listed as a community event,
+we require a few things.
+This is primarily to make sure people understand who is organizing the event,
+and to make sure companies can't run events inside our community for their own profit.
+
+Community events:
```
Comment (thatdocslady) [inline docs/organizer-guide/confs/community-events.rst:25]: Hmmmmmm I'm wondering about the name "community event", coz theoretically we run those as well. Let's talk about this at the call today, I'm thinking affiliate or something more accurate but less stuffy.

**Review by thatdocslady — COMMENTED** [review]
Awesome! Thanks for this. A few comments inline, the main question I have is about the term "community event", but we can discuss this.

File: docs/organizer-guide/confs/community-events.rst (hunk 1 above)
Comment (plaindocs) [reply docs/organizer-guide/confs/community-events.rst]: This is an arrow, not a greater than? :+1: (referring to @thatdocslady comment)

File: docs/organizer-guide/confs/community-events.rst (hunk 2 above)
Comment (plaindocs) [reply docs/organizer-guide/confs/community-events.rst]: Yeah, not entirely sure about the siblingishness. @thatdocslady makes good points.

Could just for for "related events with similar goals and values" or some such nonsense.

File: docs/organizer-guide/confs/community-events.rst (hunk 4 above)
Comment (plaindocs) [reply docs/organizer-guide/confs/community-events.rst]: s/refenence/reference/

Comment (plaindocs) [reply docs/organizer-guide/confs/community-events.rst]: > as a refenence to the community

what does this even mean? :-p

Comment (plaindocs) [reply docs/organizer-guide/confs/community-events.rst]: * Must have multiple clearly listed organizers from more than one company  

^ still not very happy with that bulletpoint

**Review by plaindocs — COMMENTED** [review]
Good starting point.

**Comment by ericholscher** [comment]
Updated this @thatdocslady @plaindocs @kelly-o if this matches about what folks are thinking?

File: docs/organizer-guide/confs/community-events.rst [hunk 6]
```diff
@@ -0,0 +1,30 @@
+Community-Run Events
+====================
+
+We are happy to promote events that are independently organized by members of our community.
+They are **not** officially organized by Write the Docs organizers,
+but they related events with similar goals and values.
```
Comment (plaindocs) [inline docs/organizer-guide/confs/community-events.rst]: missing "are"

File: docs/organizer-guide/confs/community-events.rst [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+Types of Events
+---------------
+
+In general we have a few different type of events:
```
Comment (plaindocs) [inline docs/organizer-guide/confs/community-events.rst]: Remove "In general"

File: docs/organizer-guide/confs/community-events.rst [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+------------
+
+In order to be listed as a community-run event,
+we require a few things.
+This is primarily to make sure people understand who is organizing the event,
+and to make sure companies can't run events inside our community for their own profit.
+
+Community events:
+
+* Must target our specific community
```
Comment (plaindocs) [inline docs/organizer-guide/confs/community-events.rst]: Do we need an example of what "target" means in this context?

**Review by plaindocs — APPROVED** [review]
Small nits, but otherwise LGTM

**Comment by ericholscher** [comment]
Pushed up fixes.

**Comment by plaindocs** [comment]
:ship: :it: 
@thatdocslady @kelly-o ?

**Pull request merged**